src/main/java/seedu/taassist/logic/commands/AssignCommand.java

```diff
@@ -63,8 +64,13 @@ public CommandResult execute(Model model) throws CommandException {
 
         studentsToAssign.forEach(s -> model.setStudent(s, s.addModuleClass(moduleClassToAssign)));
```

Comment:
Do we need a separate message for students who are already assigned to the class?